source/includes/SystemSettings/system_settings.md

```diff
@@ -0,0 +1,114 @@
+# System Setting
+
+## System Setting Overview
+
+* System Settings are used to store module and system-wide settings. They primarily consist of a property name and a value and a description explaining how this property is being used
```

Comment:
make sure to have .  in the end of sentences.

Reply:
@Akayeshmantha Ahh sorry I will refactor it .